### ISSUE TITLE
Add x64 platform to game projects

### ISSUE THREAD

**Issue by charabaruk** [vchelaru/FlatRedBall#136]
Support building 64-bit games with FRB.

**Comment by charabaruk** [comment]
I was able to create a x64 platform target and get a working debug build. Chances are it'll work just fine for the DebugWindows and Release configurations as well. It would probably be safe to add these to the csproj as initially generated.

Visual Studio added these property groups when I created the platform target. I changed the `OutputPath` properties to match the existing x86 groups, and removed the `XNA4` define constant in these groups (while leaving it in the existing x86 groups).

```xml
  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
    <DebugSymbols>true</DebugSymbols>
    <OutputPath>bin\$(MonoGamePlatform)\$(Platform)\$(Configuration)\</OutputPath>
    <DefineConstants>TRACE;DEBUG;MONOGAME; DESKTOP_GL; FRB_XNA;</DefineConstants>
    <DebugType>full</DebugType>
    <PlatformTarget>x64</PlatformTarget>
    <LangVersion>7.3</LangVersion>
    <ErrorReport>prompt</ErrorReport>
    <Prefer32Bit>true</Prefer32Bit>
  </PropertyGroup>
  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
    <OutputPath>bin\$(MonoGamePlatform)\$(Platform)\$(Configuration)\</OutputPath>
    <DefineConstants>TRACE;MONOGAME; DESKTOP_GL; FRB_XNA;</DefineConstants>
    <Optimize>true</Optimize>
    <DebugType>pdbonly</DebugType>
    <PlatformTarget>x64</PlatformTarget>
    <LangVersion>7.3</LangVersion>
    <ErrorReport>prompt</ErrorReport>
    <Prefer32Bit>true</Prefer32Bit>
  </PropertyGroup>
  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'DebugWindows|x64'">
    <DebugSymbols>true</DebugSymbols>
    <OutputPath>bin\x64\DebugWindows\</OutputPath>
    <DefineConstants>TRACE;DEBUG;MONOGAME; DESKTOP_GL; FRB_XNA; WINDOWS</DefineConstants>
    <DebugType>full</DebugType>
    <PlatformTarget>x64</PlatformTarget>
    <LangVersion>7.3</LangVersion>
    <ErrorReport>prompt</ErrorReport>
    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
    <Prefer32Bit>true</Prefer32Bit>
  </PropertyGroup>
```

**Comment by vchelaru** [comment]
Fixed 1199f629d714767a05d0c2d12db28eb57ab7d822
